packages/flutter/lib/src/material/about.dart

```diff
@@ -888,13 +889,23 @@ class _PackageListTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
     return Ink(
-      color: isSelected ? Theme.of(context).highlightColor : Theme.of(context).cardColor,
-      child: ListTile(
-        title: Text(packageName),
-        subtitle: Text(MaterialLocalizations.of(context).licensesPackageDetailText(numberLicenses)),
-        selected: isSelected,
-        onTap: onTap,
+      color: isSelected ? theme.highlightColor : theme.cardColor,
+      child: ListTileTheme(
+        data: ListTileTheme.of(context).copyWith(
+          titleTextStyle: theme.textTheme.titleMedium,
+          subtitleTextStyle: theme.textTheme.bodyMedium,
+          selectedColor: theme.colorScheme.primary,
+        ),
```

Comment:
Why do we think that users want values from the Theme and not from ListTileTheme here?